iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/v1/models/DeploymentMetricsApiModel.java

```diff
@@ -14,8 +15,8 @@
 
     private static final String APPLIED_METRICS_KEY = "appliedCount";
     private static final String TARGETED_METRICS_KEY = "targetedCount";
-    private static final String SUCCESEEDED_METRICS_KEY = "succeededCount";
-    private static final String FAILED_METRICS_KEY = "failedCount";
+    private static final String SUCCESEEDED_METRICS_KEY = "reportedSuccessfulCount";
```

Comment:
Doesn't the UI check for succeededCount already? Why are we changing this key? Maybe add a description to the pull request to explain what the issue/fix was.